cmd/query/check.go

```diff
@@ -53,7 +53,7 @@ func check(
 
 	response, err := fgaClient.Check(context.Background()).Body(*body).Options(*options).Execute()
 	if err != nil {
-		return nil, fmt.Errorf("failed to check due to %w", err)
+		return nil, fmt.Errorf("due to %w", err)
```

Comment:
We currently prefer to prefix the message but it leads to situations like this issue where error messages become verbose and repeated.

I wonder if we could just return the `err` here directly rather than prefix the error? We could add `due to` on line 93, but maybe that's also a little verbose and we could just add `:`  so error messages are like  

`check failed: <error_message>`?
```suggestion
		return nil, err //nolint:wrapcheck
```

Reply:
resolved in [5670b61](https://github.com/openfga/cli/pull/405/commits/5670b61ee201a7452073b65745548913ebad728a)